storio-test-common/src/test/java/com/pushtorefresh/storio/test/StorIOAssertTest.java

```diff
@@ -0,0 +1,39 @@
+package com.pushtorefresh.storio.test;
+
+import org.junit.Test;
+
+import java.util.concurrent.atomic.AtomicInteger;
+
+import rx.Observable;
+import rx.functions.Action1;
+
+import static junit.framework.Assert.fail;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.times;
+import static org.mockito.Mockito.verify;
+
+public class StorIOAssertTest {
+
+    @SuppressWarnings("unchecked")
+    @Test
+    public void assertThatObservableEmitsOncePositive() {
+        final Action1<String> testAction = mock(Action1.class);
+        final String testString = "Test string";
+        StorIOAssert.assertThatObservableEmitsOnce(Observable.just(testString), testAction);
```

Comment:
I'm not shure I follow you. 
What is the different betwen tests and other code? 
In that case `assertThatObservableEmitsOnce` is not test subject, it's just tool. 
So in other code `static import` usage instead of class references also reduces clarity.
Ofcause, if we need to call few util methods in one place - it will be way more pleasantly and shorter. But for one line.. I don't know =)